### PR TITLE
Upgrade to openapi v3 and fixes 2019.10 specification

### PR DESCRIPTION
**Pull request by colin-swu**
## Description

* Upgrades the generation to openapi version 3.
* Fixes a parameter in /templates and /templates/:id paths (drafts previously was provided).
* Adds tests using GitHub actions.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: api_localizations.go [hunk 1]
```diff
@@ -28,14 +27,15 @@ var (
...
-Returns a specific localization object with a matching ID.
+GetLocalizationById Get Localization Object by ID
+Returns a specific localization object with a matching ID
  * @param ctx _context.Context - for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
  * @param localizationId A localization ID
  * @param targetLanguage The type of templating language to compile as. Should only be used for visual templates.
+ * @param accept A version of the API that should be used for the request. For example, to use version \"2019.10\", set the value to \"application/vnd.dyspatch.2019.10+json\"
 @return LocalizationRead
 */
-func (a *LocalizationsApiService) LocalizationsLocalizationIdGet(ctx _context.Context, localizationId string, targetLanguage string) (LocalizationRead, *_nethttp.Response, error) {
```
Comment (dlsteuer) [inline api_localizations.go:38]: this old naming is terrible

File: mod_test.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+const version = "application/vnd.dyspatch.2019.10+json"
+
+func bodyToString(body io.ReadCloser) string {
+	buf := new(bytes.Buffer)
+	buf.ReadFrom(body)
+	return buf.String()
+}
+
+func TestMain(t *testing.T) {
```
Comment (dlsteuer) [inline mod_test.go:23]: are these generated tests, or manual?

**Review by dlsteuer — APPROVED** [review]
Seems good, if the tests are created manually I'd suggest some changes like maybe remove the print stuff, and maybe use testify requires instead of t.Fatal, but not a big deal.

File: mod_test.go (hunk 2 above)
Comment (colin-swu) [reply mod_test.go:23]: These are manual, ya I definitely could clean some of them up copied some code Dylan had into here.

Comment (dlsteuer) [reply mod_test.go:23]: up to you, would be a nice improvement is all.

Comment (colin-swu) [reply mod_test.go:23]: I'll add a task into the backlog for API client tests. Just to push along the work at the moment.

**Pull request merged**